activiti-core/activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/model/Extension.java

```diff
@@ -82,18 +83,22 @@ public VariableDefinition getPropertyByName(String name) {
         return null;
     }
 
-    public boolean hasEmptyInputsMapping(String elementId) {
-        ProcessVariablesMapping processVariablesMapping = mappings.get(elementId);
-        return processVariablesMapping != null && processVariablesMapping.getInputs().size() == 0;
+    public boolean hasMapping(String taskId) {
+        return mappings.get(taskId) != null;
     }
 
-    public boolean hasEmptyOutputsMapping(String elementId) {
+    public boolean hasMappingTypeInputs (String elementId) {
```

Comment:
I'd rename it to `shouldMapAllInputs`.

Reply:
I wouldn't name something that returns a boolean with `should` prefix, I believe `has`, `is` is more appropriate for those

Reply:
Yes, now makes sense. It's difficult now to change, we should avoid to have something like this if possible as makes it hard to understand what's going on

Reply:
let's rename it and also remove that tailing space as well please @miguelruizdev